src/fides/api/models/property.py

```diff
@@ -41,6 +43,9 @@ def __tablename__(self) -> str:
         primary_key=True,
         default=generate_id,
     )
+    # Right now, we use server default to write this val.
+    # In the future we may allow ability to configure which property is the default
+    is_default = Column(Boolean, server_default="f", default=False)
```

Comment:
todo- Add logic to adding a property, where if it's the only property, it's the default property. We need something that will ensure there's always a default property in advance of being able to provide this toggle in the UI